reference/forms/types/datetime.rst

```diff
@@ -161,6 +162,13 @@ this format.
 
 .. include:: /reference/forms/types/options/seconds.rst.inc
 
+input_format
```

Comment:
I think we should move this description directly below the description of the `input` option.

Reply:
Definitely! That was a mistake.